### PR TITLE
Update clang-tidy.md

### PR DESCRIPTION
**Pull request by Omotola**
Added information for a new project property that allows selection of a custom clang-tidy exe

### CODE REVIEW

**Comment by Court72** [comment]
@omotola,

When this PR is ready for review, remove **Draft** mode by selecting **Ready for review**.

image

**Comment by Court72** [comment]
@TylerMSFT 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/code-quality/clang-tidy.md [hunk 1]
```diff
@@ -90,6 +90,10 @@ Clang-Tidy runs result in warnings displayed in the Error List, and as in-editor
...
 
+## Clang-Tidy Tool Directory
+
+If you'd like to have custom rules built into your clang-tidy executable and run it in VS, you can change the path to the executable that Visual Studio runs. In the project Property Pages dialog, open the **Configuration Properties** > **Code Analysis** > **Clang-Tidy** page. Manually type in the path or Browse and select the path under the **Clang-Tidy Tool Directory** property. Once the change is saved, the new executable will be used after the next succesful compilation.
```
Comment (TylerMSFT) [inline docs/code-quality/clang-tidy.md:95]: VS - we have to spell it out (legal). We put Microsoft Visual Studio on first usage.

Comment (TylerMSFT) [inline docs/code-quality/clang-tidy.md:95]: Should Browse be in bold? (Are you referring to a button? If so, bold)
Avoid future tense, e.g. "The new executable is used once the change is saved and the app is recompiled."

**Review by TylerMSFT — CHANGES_REQUESTED** [review]
A couple minor suggestions.

**Review by TylerMSFT — APPROVED** [review]
Thank you for updating this topic

**Review by TylerMSFT — APPROVED** [review]

File: docs/code-quality/clang-tidy.md [hunk 2]
```diff
@@ -90,6 +90,10 @@ Clang-Tidy runs result in warnings displayed in the Error List, and as in-editor
...
 
+## Clang-Tidy Tool Directory
+
+If you'd like to have custom rules built into your clang-tidy executable and run it in Microsoft Visual Studio, you can change the path to the executable that Visual Studio runs. In the project Property Pages dialog, open the **Configuration Properties** > **Code Analysis** > **Clang-Tidy** page. Manually type in the path or **Browse** and select the path under the **Clang-Tidy Tool Directory** property. Once the change is saved, the new executable is used once the change is saved and the app is recompiled.
```
Comment (Jak-MS) [inline docs/code-quality/clang-tidy.md:95]:
```suggestion
If you'd like to have custom rules built into your clang-tidy executable and run it in Microsoft Visual Studio, you can change the path to the executable that Visual Studio runs. In the project Property Pages dialog, open the **Configuration Properties** > **Code Analysis** > **Clang-Tidy** page. Manually type in the path or **Browse** and select the path under the **Clang-Tidy Tool Directory** property. The new executable is used once the change is saved, and the app is recompiled.
```
removing the redundant "once the change is saved"

**Review by Jak-MS — APPROVED** [review]

**Pull request merged**